### PR TITLE
Projects redesign/font size

### PR DESCRIPTION
**Pull request by sunilsabatp**
Points covered in this PR :

- Changed the position of the "Explore" button and updated its font size
- Updated the font size in the Project Info section

### CODE REVIEW

**Review by mariahosfeld — DISMISSED** [review]
Please increase the font size of the highlighted (green) element from 8px to 10px
image

File: src/features/projectsV2/ProjectDetails/styles/AboutProject.module.scss [hunk 1]
```diff
@@ -9,16 +9,15 @@
   button {
     display: flex;
     gap: 6px;
-    font-size: 10px;
     font-weight: 700;
+    font-size: inherit;
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/styles/AboutProject.module.scss:13]: Should we leave the "Read More" button font size as 10px?

File: src/features/projectsV2/ProjectDetails/styles/ProjectInfo.module.scss [hunk 2]
```diff
@@ -140,6 +141,9 @@
   gap: 6px;
   align-items: center;
   min-width: 50%;
+  a {
+    font-size: $fontXXSmall;
+  }
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/styles/ProjectInfo.module.scss:146]: Please add a class selector for the styles applied for the year, it keeps things clearer.

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@sunilsabatp please review my feedback

**Review by mohitb35 — APPROVED** [review]
LGTM

**Pull request merged**